main.tf

```diff
@@ -87,17 +88,16 @@ data "aws_iam_policy_document" "assume_from_scalr" {
       variable = "${var.scalr_hostname}:sub"
       values = [
         format(
-          "account:%s:environment:%s:workspace:*",
-          var.scalr_account_name,
-          var.scalr_environment_name,
+          "scalr:account:%s",
+          var.scalr_account_name
         )
       ]
     }
   }
 }
 
```

Comment:
The `server_side_encryption_configuration` block is deprecated in the main S3 bucket resource. It should be moved to a separate `aws_s3_bucket_server_side_encryption_configuration` resource.